src/javascripts/Services/mainTreeElementService.js

```diff
@@ -125,6 +126,11 @@ ngapp.service('mainTreeElementService', function(editModalFactory, errorService,
             errorService.try(() => scope.$emit('openModal', 'edit', modalOptions));
         };
 
+        scope.addMasters = function(node) {
+            let modalOptions = node;
+            errorService.try(() => scope.$emit('openModal', 'addMasters', modalOptions));
```

Comment:
**(Note for myself)**
I need to re-evaluate why I've been using `errorService.try` when emitting openModal.  It shouldn't be necessary because there's no exception that could happen in the [openModal listener](https://github.com/matortheeternal/zedit/blob/master/src/javascripts/Views/base.js#L65-L72) (it won't be catching anything from the modal's initialization, or even from the option building).